modules/audio_splitter.py

```diff
@@ -0,0 +1,37 @@
+from pydub import AudioSegment
+from pydub.silence import split_on_silence
+import os
+import time
+
+def split_audio(audio_path, output_path, min_silence_len=500, silence_thresh=-42):
+    """
+    split an audio into chunks by a silence threshold
+
+    :param audio_path: path to audio (any format that ffmpeg supports)
+    :param output_path: chunks output path
+    :param min_silence_len: (in ms) minimum length of a silence to be used for
+        a split. default: 500ms
+    :param silence_thresh: (in dBFS) anything quieter than this will be
+        considered silence. default: -42dBFS
+    """
+
+    file_name = os.path.basename(audio_path).split('.')[0]
```

Comment:
O que acha dessa estratégia?
```suggestion
    file_name = os.path.basename(audio_path).split('.').pop()
```